manager/integration/tests/test_statefulset.py

```diff
@@ -33,6 +53,59 @@ def create_and_wait_statefulset(statefulset_manifest):
     wait_statefulset(statefulset_manifest)
 
 
+def create_and_test_backups(api, cli, pod_data):
```

Comment:
`create_and_test_pod_backups`.

And comments to help on what's `pod_data` here?